permissions_validators/src/public_blockchain/mod.rs

```diff
@@ -42,6 +42,7 @@ pub fn default_permissions<W: WorldTrait>() -> IsInstructionAllowedBoxed<W> {
     ValidatorBuilder::new()
         .with_recursive_validator(grant_instruction_validator)
         .with_recursive_validator(transfer::OnlyOwnedAssets.or(transfer::GrantedByAssetOwner))
+        .with_recursive_validator(transfer::ExecutionCountFitsInLimit)
```

Comment:
I think this shouldn't be the default. Restricting transfers is specific for the Orillion use-case.